.github/workflows/e2e.yml

```diff
@@ -20,3 +20,9 @@ jobs:
       - run: |
           npx playwright test
         working-directory: ./src/test/e2e
+      - if: ${{ always() }}
```

Comment:
Is it needed? Seems always true?

Reply:
yes, we need these result no matter the upper step failed or success.
in github, once the front steps failed, policy is fast-fail and skip all the subsequent steps